pkg/markers/parse.go

```diff
@@ -84,6 +84,8 @@ const (
 	InvalidType ArgumentType = iota
 	// IntType is an int
 	IntType
+	// NumberType is a float64
+	NumberType
```

Comment:
Wouldn't the same `FloatType` make more sense here? Or is there are reason for this?

Reply:
These names reflect the schema-level type names and not the go-level names; in the CRD output this will be "type: number".

Reply:
That's what the json schema calls it

Reply:
Thanks for the explanation.